deploy/gcp/README.md

```diff
@@ -71,25 +124,41 @@ kubectl get po -n tidb
 helm ls
 ```
 
-When done, the infrastructure can be torn down by running `terraform destroy`
+## Upgrade
 
+To upgrade the TiDB cluster, modify the `tidb_version` variable to a higher version in `variables.tf` and run `terraform apply`.
 
-## Upgrade TiDB cluster
+For example, to upgrade the cluster to the 2.1.10 version, modify the `tidb_version` to `v2.1.10`:
 
-To upgrade TiDB cluster, modify `tidb_version` variable to a higher version in variables.tf and run `terraform apply`.
+```
+variable "tidb_version" {
+description = "tidb cluster version"
+default = "v2.1.10"
+}
+```
 
-> *Note*: The upgrading doesn't finish immediately. You can watch the upgrading process by `watch kubectl --kubeconfig credentials/kubeconfig_<cluster_name> get po -n tidb`
+The upgrading does not finish immediately. You can watch the upgrading process by `kubectl --kubeconfig credentials/kubeconfig_<cluster_name> get po -n tidb --watch`.
 
-## Scale TiDB cluster
+## Scale
 
-To scale TiDB cluster, modify `tikv_count`, `tikv_replica_count`, `tidb_count`, and `tidb_replica_count` to your desired count, and then run `terraform apply`.
+To scale the TiDB cluster, modify `tikv_count`, `tikv_replica_count`, `tidb_count`, and `tidb_replica_count` to your desired count, and run `terraform apply`.
 
-> *Note*: Currently, scaling in is not supported since we cannot determine which node to remove. Scaling out needs a few minutes to complete, you can watch the scaling out by `watch kubectl --kubeconfig credentials/kubeconfig_<cluster_name> get po -n tidb`
+Currently, scaling in is not supported since we cannot determine which node to remove. Scaling out needs a few minutes to complete, you can watch the scaling-out process by `kubectl --kubeconfig credentials/kubeconfig_<cluster_name> get po -n tidb --watch`.
+
+For example, to scale out the cluster, you can modify the number of TiDB instances from 2 to 3:
+
+```
+variable "tidb_count" {
+default = 3
+}
+```
 
-> *Note*: Incrementing the node count will create a node per GCP availability zones.
+> *Note*: Incrementing the node count will create a node per GCP availability zone.
```

Comment:
Should be "available zone"?

Reply:
In the `variables.tf` file, the descriptions for `tidb_count`, `tikv_count` and other variables all use "per GCP availability zone", so I think we should unify the wording by using "availability" here.